src/MeetingsSDKAdapter.js

```diff
@@ -131,11 +131,13 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
   /**
    * Update the meeting object by removing media based on a given event type.
    *
-   * @param {string} ID  ID of the meeting to fetch
+   * @param {string} ID     ID of the meeting to fetch
+   * @param {Object} event  Event type that was triggered
+   * @param {Subject} end$  Subject used to complete @see {@link getMeeting}
    * @memberof MeetingsSDKAdapter
    * @private
    */
-  removeMedia(ID, {type}) {
+  removeMedia(ID, {type}, end$) {
```

Comment:
no switch case needed here anymore. for now, `MEDIA:STOPPED` gets only triggered if we are leaving the meeting. Therefore, we should remove all attributes from the existing meeting object except the ID. to just keep record of it. NOTE: the ID is SDK MEETING ID which will be removed after leaving the meeting from the meetings collections

Reply:
Removed switch case